core/src/main/resources/hudson/widgets/HistoryWidget/index.jelly

```diff
@@ -23,76 +23,52 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+               text="Atom feed ${%for all}"
+               href="${it.baseUrl}/rssAll" />
+      <dd:item icon="symbol-rss"
+               text="Atom feed ${%for failures}"
+               href="${it.baseUrl}/rssFailed" />
+    </l:overflowButton>
+  </j:set>
 
-  <j:set var="page" value="${it.historyPageFilter}" />
-  <div id="buildHistoryPage" page-ajax="${it.baseUrl}/buildHistory/ajax" page-entry-newest="${page.newestOnPage}" page-entry-oldest="${page.oldestOnPage}" page-has-up="${page.hasUpPage}" page-has-down="${page.hasDownPage}">
-    <div id="buildHistoryPageNav">
-      <div class="buildHistoryPageNav__item buildHistoryPageNav__item--page-one pageOne" title="Page 1 (Newest/Latest Builds)">
-        <div class="buildHistoryPageNav__item-page-one-top"></div>
-        <l:svgIcon href="${resURL}/images/svgs/go-up.svg#arrow" viewBox="0 0 16 16" />
-      </div>
-      <div class="buildHistoryPageNav__item pageUp" title="Newer Builds">
-        <l:svgIcon href="${resURL}/images/svgs/go-up.svg#arrow" viewBox="0 0 16 16" />
-      </div>
-      <div class="buildHistoryPageNav__item pageDown" title="Older Builds">
-        <l:svgIcon href="${resURL}/images/svgs/go-down.svg#arrow" viewBox="0 0 16 16" />
-      </div>
-    </div>
+  <div id="buildHistoryPage" page-ajax="${it.baseUrl}/buildHistory/ajax"
+       data-page-entry-newest="${page.newestOnPage}"
+       data-page-entry-oldest="${page.oldestOnPage}"
+       data-page-has-up="${page.hasUpPage}"
+       data-page-has-down="${page.hasDownPage}">
+    <l:card id="jenkins-builds" title="Builds" controls="${controls}" expandable="buildTimeTrend">
+      <l:search-bar placeholder="${%find}"
+                    clazz="${page.runs.isEmpty() and page.queueItems.isEmpty() ? 'jenkins-hidden' : ''}"/>
 
-    <l:pane width="3" title="${h.runScript(paneTitle)}" footer="${h.runScript(paneFooter)}" id="buildHistory" class="jenkins-pane stripped">
-      <tr class="build-search-row">
-        <td>
-          <l:search-bar placeholder="${%find}" clazz="${page.runs.isEmpty() and page.queueItems.isEmpty() ? 'jenkins-hidden' : ''}"/>
-          <div id="no-builds" class="jenkins-pane__information" style="${page.runs.isEmpty() and page.queueItems.isEmpty() ? '' : 'display: none;'}">
-            ${%No builds}
-          </div>
-        </td>
-      </tr>
+      <div class="app-builds-container">
+        <div id="no-builds" class="app-builds-container__placeholder">
+          ${%No builds}
+        </div>
 
-      <st:include page="entries.jelly" it="${page}" />
+        <div id="jenkins-build-history" class="app-builds-container__items">
+        </div>
 
-    </l:pane>
+        <div class="app-builds-container__controls" id="controls">
+          <button class="jenkins-button jenkins-button--tertiary jenkins-card__unveil" id="up">
+            <l:icon src="symbol-arrow-left" />
+          </button>
+          <button class="jenkins-button jenkins-button--tertiary jenkins-card__unveil" id="down">
```

Comment:
These need hidden labels or tooltips or something to indicate what they do, there's no a11y information on them currently

Reply:
Updated with some hidden spans, content is currently 'Newer builds' and 'Older builds'